rest_framework/routers.py

```diff
@@ -130,19 +130,13 @@ def get_default_base_name(self, viewset):
         If `base_name` is not specified, attempt to automatically determine
         it from the viewset.
         """
-        # Note that `.model` attribute on views is deprecated, although we
-        # enforce the deprecation on the view `get_serializer_class()` and
-        # `get_queryset()` methods, rather than here.
-        model_cls = getattr(viewset, 'model', None)
         queryset = getattr(viewset, 'queryset', None)
-        if model_cls is None and queryset is not None:
-            model_cls = queryset.model
 
-        assert model_cls, '`base_name` argument not specified, and could ' \
+        assert queryset, '`base_name` argument not specified, and could ' \
```

Comment:
Does `assert queryset is not None` change anything here - are we accidentally evaluating the queryset?